src/main/java/com/example/demo/model/mapper/HealthcareFacilityMapper.java

```diff
@@ -16,6 +16,8 @@
 
 @Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public interface HealthcareFacilityMapper {
+    HealthcareFacilityMapper INSTANCE = Mappers.getMapper(HealthcareFacilityMapper.class);
```

Comment:
@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
Mamy tego mappera jako beana wiec po co nam ten INSTANCE mozemy go sobie wstrzykiwac a jesli nie chcemy wstrzykiwac to wezmy sobie poprzez Mappers.getMapper(HealthcareFacilityMapper.class)